AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -175,6 +176,15 @@ - (void)setBackgroundColor:(UIColor *)backgroundColor
   _placeholderTextKitComponents.textView.backgroundColor = backgroundColor;
 }
 
+- (void)setTextContainerInset:(UIEdgeInsets)textContainerInset
+{
+  ASDN::MutexLocker l(_textKitLock);
+
+  _textContainerInset = textContainerInset;
+  _textKitComponents.textView.textContainerInset = textContainerInset;
```

Comment:
Would accessing textView here cause the view to be created?  Naturally that would be bad...it may be best to wrap the last two lines in a check of [self isNodeLoaded] (I may have the exact property name wrong)